cli/src/main/java/org/jboss/as/cli/operation/impl/DefaultCallbackHandler.java

```diff
@@ -625,7 +625,13 @@ public int getLastSeparatorIndex() {
 
     @Override
     public int getLastChunkIndex() {
-        return lastChunkIndex;
+        int originalLineLength = originalLine.length();
+        int substitutedLineLength = substitutedLine.length();
+        if (originalLineLength == substitutedLineLength)
+            return lastChunkIndex;
+        else {
+            return lastChunkIndex - (substitutedLineLength - originalLineLength);
+        }
```

Comment:
Would you mind changing it to a single line? I.e.
return lastChunkIndex - substitutedLine.length + originalLine.length();